sdk/test-utils/recorder-new/src/utils/relativePathCalculator.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import path from "path";
+import fs from "fs";
+import { RecorderError } from "./utils";
+
+export function relativeRecordingsPathForNode(testContext: Mocha.Test) {
```

Comment:
There is a builtin called `path.relative` that may replace this function.

Reply:
Good suggestion, applied